public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico?v=3" />
```

Comment:
I noticed that your favicon file was in png. If you wanted to add it as a favicon on the top browser bar, I believe it has to be a .ico file.